files/templates/additional-scrape-configs.template.yaml

```diff
@@ -826,12 +855,12 @@
     insecure_skip_verify: true
 {{ end }}
   relabel_configs:
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring]
+  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring;__meta_kubernetes_service_labelpresent_giantswarm_io_monitoring]
```

Comment:
```suggestion
  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring, __meta_kubernetes_service_labelpresent_giantswarm_io_monitoring]
```

Reply:
I tried that before. Let me change it back so you can try to deploy it

Reply:
Done

Reply:
Cool, thanks.

So this is on managed app target, so to test it I need to create a TC, then deploy some managed app and check if it's picked up, right? Which managed app did you use for testing BTW?

Reply:
It worked with externaldns, a default app

Reply:
You can always deploy your app with the label and annotation

Reply:
OK, but what app? Bear with me here, I've never deployed an app and have no clue how you even do that yet.

Anyway, I'll try with externaldns sure.

Reply:
I was thinking you could deploy any kubernetes deployment, crearte a service that points to the metrics path ahd add the label and annotations htere but otherwise, you can do it via happ or by creating an app CR

Reply:
Nginx, the latest 1.9.2 version can also be installed

Reply:
I tried with nginx-IC and I think I got it, but there's already a separate target specifically for nginx-IC and it's being picked up there.

OK, I'll try something else.